### PR TITLE
callbacks example

### PR DESCRIPTION
**Pull request by ArinaDanilina**

### CODE REVIEW

**Comment by MUCDK** [comment]
please provide a custom callback example. e.g. take ATAC counts and do LSI (standard preprocessing for ATAC data)

File: examples/problems/1100_callback.ipynb [hunk 1]
```diff
@@ -0,0 +1,502 @@
+{
```
Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>add a header above as you do for the spatial normalisation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>add a header above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>I think therefore we have a different tutorial, right? then just link that example here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (ArinaDanilina) [reply examples/problems/1100_callback.ipynb:1]: this is the example for using graphs too

File: examples/problems/1100_callback.ipynb [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
```
Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>PCA computation in gene space.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>we don't get a joint cost. we get a joint embedding.</p><p> </p><p>Say "To create a joint PCA embedding between two sets of genes, we compute the PCA embedding for pairs of distributions by passing [and now what you said]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>Please add a description above. something along the lines that we have two point clouds living in PCA space.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>Using geodesic costs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>To use geodesic costs defined on a graph, we can create the underlying graph (here in gene expression space) using [...]. Note that the <code>cost</code> has to be set explicitly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>Can you please create a link to the API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>Say above that we prepare the problem with the sq. euc. cost (default), but then overwrite it by using <code>set_graph_xy</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (MUCDK) [inline examples/problems/1100_callback.ipynb:1]: <p>why do we need the term? i.e. lambda term, src, tgt?</p><p> </p><p>And please provide a description about the signature of the callbacks. i.e. input types and output types, in the description above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Review by MUCDK — APPROVED** [review]
Thanks Arina!

**Pull request merged**